pokedex/data/csv/versions.csv

```diff
@@ -33,3 +33,4 @@ id,version_group_id,identifier
 32,19,lets-go-eevee
 33,20,sword
 34,20,shield
+35,20,legends-arceus
```

Comment:
Hi! Sorry for coming back only now. I think the PR looks good, but I'd like to request a change/addition. 

In between _shield_ and _legends-arceus_ there are other 4 versions that we didn't add. We added the pokemon released in those 4 versions but we didn't actually add the versions.

Could you insert the versions:
- The Isle of Armor
- The Crown Tundra
- Brilliant Diamond
- Shining Pearl

And the version groups: 
- The Isle of Armor
- The Crown Tundra
- Brilliant Diamond and Shining Pearl

Thanks so much! I think you'll need also to update the other files you contributed to, such as `pokemon_forms.csv`

Reply:
Sure, I will check it and update.

Is that 4 version are same group? How about the Legends Arceus, same or another one?

Reply:
So, there are 4 new groups to be added:

```
the-isle-of-armor
the-crown-tundra
brilliant-diamond-and-shining-pearl
legends-arceus
```

and there are 5 new versions:

```
the-isle-of-armor
the-crown-tundra
brilliant-diamond
shining-pearl
legends-arceus
```